filters/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+add_library(filters INTERFACE)
+
+add_library(${PROJECT_NAME}::FILTER ALIAS filters)
```

Comment:
I think we have the same name in caps in all other subdirs. I am not sure what effect this has.

Reply:
I'm not entirely sure either, tbh. I think the official name is `datasketches::FILTER` and we alias it to `filter`?

Reply:
should this be the same as the subdir?

Reply:
missing an S? probably? but only if we also put quotient and infinifilter in there.  otherwise this should probably live elsewhere.

Reply:
I think we will put quotient filter there. and eventually infinifilter

Reply:
still not sure it's actually needed or even best practice, but added the S